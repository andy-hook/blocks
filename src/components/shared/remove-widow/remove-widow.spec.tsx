import React from "react"

import RemoveWidow from "./remove-widow"
import { renderWithTheme } from "@test-utils"

describe("<RemoveWidow />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(
      "dark",
      <RemoveWidow>This is test text</RemoveWidow>
    )

    expect(tree).toMatchSnapshot()
  })
})
