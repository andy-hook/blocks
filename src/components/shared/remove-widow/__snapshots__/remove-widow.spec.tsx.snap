// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RemoveWidow /> renders correctly 1`] = `"This is testÂ text"`;
